ibis/expr/operations/arrays.py

```diff
@@ -266,3 +266,60 @@ class TimestampRange(Range):
     start: Value[dt.Timestamp]
     stop: Value[dt.Timestamp]
     step: Value[dt.Interval]
+
+
+class ArrayAgg(Value):
+    arg: Value[dt.Array]
+    shape = rlz.shape_like("args")
+
+    @attribute
+    def dtype(self) -> dt.DataType:
+        return self.arg.dtype.value_type
+
+
+@public
+class ArrayMin(ArrayAgg):
+    """Compute the minimum value of an array."""
+
+
+@public
+class ArrayMax(ArrayAgg):
+    """Compute the maximum value of an array."""
+
+
+# in duckdb summing an array of ints leads to an int, but for other backends
+# it might lead to a float??
+@public
+class ArraySum(ArrayAgg):
```

Comment:
Should this also work on arrays of bools? Could punt to a followup if someone actually requests it

Reply:
I think we can wait until someone requests it.